packages/gatsby/src/utils/cache-lmdb.ts

```diff
@@ -16,9 +16,12 @@ export default class GatsbyCacheLmdb {
   private static store
   private db: Database | undefined
   public readonly name: string
+  // Needed for plugins that want to write data to the cache directory
+  public readonly directory: string
 
   constructor({ name = `db` }: { name: string }) {
     this.name = name
+    this.directory = path.join(process.cwd(), `.cache/caches/${name}`)
```

Comment:
Oh, actually - do we need to also ensure this dir exists like in https://github.com/gatsbyjs/gatsby/blob/a0b542be35c15e71f8fe76cc5400576dfd0b1391/packages/gatsby/src/utils/cache.ts#L33 ?

Reply:
Yeah, that's a good point. We probably should ensure that

Reply:
@pieh added an `init` method